docs/manage/data/label.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Label Data"
+linkTitle: "Label Data"
+description: "Label image data on the DATA page in the Viam Cloud."
+weight: 39
+type: "docs"
+tags: ["data management", "cloud", "sync"]
+images: [ "/manage/data/img/label-dog.gif" ]
+# SME: Tahiya Salam and Alexis Wei
+---
+
+To label data in a dataset, go to the [**DATA** tab](https://app.viam.com/data/view) in the Viam app.
+
+On the **Images** subtab, you can filter available images, using the **Filtering** menu and select the attributes that match where, how, and when the data was collected.
+
+{{< alert title="Important" color="note" >}}
+Datasets are views, not materialized.
+If the underlying data matching the filter changes because data is deleted or more data is added, the dataset will also change.
+
+Therefore the data you are viewing may change as you label and train on the dataset.
+{{< /alert >}}
+
+You can label you data with
+
+- [Bounding boxes](#bounding-boxes), which you can use to [train object detection models](/manage/ml/train-model/#train-a-model).
+- [Image tags](#image-tags), which you can use to [train object detection models](/manage/ml/train-model/#train-a-model).
+
+### Bounding boxes
+
+You can create bounding boxes for objects in your images, which then allow you to [create object detection models](/manage/ml/train-model/#train-a-model).
+For example, if you would like to create a model that detects a dog in an image, add bounding boxes around the dog in each of your images and add or select the label `dog`.
+
+To add a bounding box, click on the image you want to add the bounding box.
+More information about the image will open up to the right side.
+Select the **Bounding Box** option.
+Choose a label or create a new label to choose and then click on the image where you would like to add the bounding box and drag to where the bounding box should end.
+
+{{<gif webm_src="../img/label-dog.webm" mp4_src="../img/label-dog.mp4" alt="Add bounding box label around dog in an image">}}
+
+Repeat this with all images in your dataset.
+
+To delete a bounding box, click the cross next to the corresponding label.
+
+### Image Tags
+
+You can use tags to [create classification models](/manage/ml/train-model/#train-a-model) for images.
+For example, if you would like to create a model that identifies an image of a star in a set of images, tag each image with a star with a `star` tag.
+The filter also needs to include images without the star tag or with another tag like `notstar`.
+If you add a `nostar` tag, you can filter the data in your dataset by adding the explicit tags `star` and `notstar` as a filter.
+Alternatively you can use date ranges to filter for relevant data.
+
+To tag an image, click on the image.
+More information about the image will open up to the right side.
+Select the **Image Tags** option.
+
+To add a bounding box, click on the image you want to add the bounding box to and, in the menu that opens, select the Bounding Box option.
```

Comment:
We can delete the information about "To add a bounding box, click on the image you want to add the bounding box to and, in the menu that opens, select the Bounding Box option." since this is specific to just bounding boxes, while this section is about Image Tags